### ISSUE TITLE
Allow text as a layer

### ISSUE THREAD

**Issue by Daniiltje** [drewwilson/atvImg#18]
Would be nice to allow the user to add a layer with text. Or is this already possible? Because it seems to empty the layer if I add text in a div with the class "atvImg-layer"

**Comment by simsketch** [comment]
Right, and that happens because it is grabbing the value from the `data-img` attribute. It's something that could be built in but is not presently supported. I can figure it out but it doesn't look like he's been very active, so not sure if the pull request would make it in...but we can always try I suppose!

**Comment by liquidvisual** [comment]
I'd really like to see this too
